diktat-rules/src/test/resources/test/paragraph3/spaces/WhiteSpaceBeforeLParExpected.kt

```diff
@@ -15,3 +15,8 @@ class Example : SuperExample {
         when (expression) { }
     }
 }
+
+data class Example(
```

Comment:
same bug was with functions?
now it fixes `foo ()`?

Reply:
Thanks, this case was missing in tests. Added.
But diktat fixes it already.